### PR TITLE
adding admin and pilot tags

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -90,41 +90,41 @@ dirac =
...
+    dirac-admin-voms-sync = DIRAC.ConfigurationSystem.scripts.dirac_admin_voms_sync:main [admin]
+    dirac-configuration-cli = DIRAC.ConfigurationSystem.scripts.dirac_configuration_cli:main [admin]
+    dirac-configuration-dump-local-cache = DIRAC.ConfigurationSystem.scripts.dirac_configuration_dump_local_cache:main [admin]
+    dirac-configuration-shell = DIRAC.ConfigurationSystem.scripts.dirac_configuration_shell:main [admin]
     # Core
-    dirac-agent = DIRAC.Core.scripts.dirac_agent:main [server]
+    dirac-agent = DIRAC.Core.scripts.dirac_agent:main [server,pilot]
     dirac-configure = DIRAC.Core.scripts.dirac_configure:main
     dirac-executor = DIRAC.Core.scripts.dirac_executor:main [server]
     dirac-externals-requirements = DIRAC.Core.scripts.dirac_externals_requirements:main
```
Comment (chrisburr) [inline setup.cfg]: I think this one will become useless once we're only using DIRACOS 2? (maybe worth leaving commented so we remember to clean it up in v8)
```suggestion
    # dirac-externals-requirements = DIRAC.Core.scripts.dirac_externals_requirements:main
```

File: setup.cfg [hunk 2]
```diff
@@ -147,67 +147,67 @@ console_scripts =
...
+    dirac-admin-users-with-proxy = DIRAC.FrameworkSystem.scripts.dirac_admin_users_with_proxy:main [admin]
     dirac-install-component = DIRAC.FrameworkSystem.scripts.dirac_install_component:main [server]
     dirac-install-tornado-service = DIRAC.FrameworkSystem.scripts.dirac_install_tornado_service:main [server]
-    dirac-monitoring-get-components-status = DIRAC.FrameworkSystem.scripts.dirac_monitoring_get_components_status:main
+    dirac-monitoring-get-components-status = DIRAC.FrameworkSystem.scripts.dirac_monitoring_get_components_status:main [admin]
     dirac-myproxy-upload = DIRAC.FrameworkSystem.scripts.dirac_myproxy_upload:main
-    dirac-populate-component-db = DIRAC.FrameworkSystem.scripts.dirac_populate_component_db:main
-    dirac-proxy-destroy = DIRAC.FrameworkSystem.scripts.dirac_proxy_destroy:main
+    dirac-populate-component-db = DIRAC.FrameworkSystem.scripts.dirac_populate_component_db:main [admin]
+    dirac-proxy-destroy = DIRAC.FrameworkSystem.scripts.dirac_proxy_destroy:main [admin]
```
Comment (chrisburr) [inline setup.cfg]: Why admin only?
```suggestion
    dirac-proxy-destroy = DIRAC.FrameworkSystem.scripts.dirac_proxy_destroy:main
```

File: setup.cfg [hunk 3]
```diff
@@ -90,41 +90,41 @@ dirac =
...
+    dirac-agent = DIRAC.Core.scripts.dirac_agent:main [server,pilot]
     dirac-configure = DIRAC.Core.scripts.dirac_configure:main
     dirac-executor = DIRAC.Core.scripts.dirac_executor:main [server]
     dirac-externals-requirements = DIRAC.Core.scripts.dirac_externals_requirements:main
     dirac-info = DIRAC.Core.scripts.dirac_info:main
-    dirac-install-db = DIRAC.Core.scripts.dirac_install_db:main
+    dirac-install-db = DIRAC.Core.scripts.dirac_install_db:main [server]
     dirac-install-web-portal = DIRAC.Core.scripts.dirac_install_web_portal:main [server]
-    dirac-platform = DIRAC.Core.scripts.dirac_platform:main
+    dirac-platform = DIRAC.Core.scripts.dirac_platform:main [pilot]
```
Comment (atsareg) [inline setup.cfg]: I would leave it for a normal user client. It can be useful for a user to know on which platform he compiles his executable.

**Comment by fstagni** [comment]
Review OK here? @chaen @atsareg

**Pull request merged**